exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
```

Comment:
Good. Is there another solution to this? Could we have called isNegative(number) inside the function isBetweenZeroAnd10(number)?